rusoto/core/src/signature.rs

```diff
@@ -503,6 +503,7 @@ fn build_hostname(service: &str, region: &Region) -> String {
                     remove_scheme_from_custom_hostname(endpoint).to_owned()
                 }
                 Region::CnNorth1 => format!("{}.{}.amazonaws.com.cn", service, region.name()),
+                Region::CnNorthwest1 => format!("{}.{}.amazonaws.com.cn", service, region.name()),
```

Comment:
Can we add these inline?

`Region::CnNorth1 | Region::CnNorthwest1 => format!("{}.{}.amazonaws.com.cn", service, region.name()),`

One example of us currently doing that: https://github.com/rusoto/rusoto/blob/master/service_crategen/src/service.rs#L113